database/models/Site.ts

```diff
@@ -5,13 +5,21 @@ import {
   Table,
   CreatedAt,
   UpdatedAt,
+  DeletedAt,
   BelongsToMany,
+  HasOne,
+  BelongsTo,
+  ForeignKey,
 } from "sequelize-typescript"
 
+import { SiteStatus, JobStatus } from "../../constants"
+
```

Comment:
Nit: we could use `@constants/index` instead

Reply:
This nit is actually a slightly bigger fix; fixed in 5a4eca3.

### Description of fix

To get this to work, I got into a mini-rabbit hole. 

I was curious as to why our CMS backend codebase doesn't use the aliases in the `database/` initially - then I saw this comment in `tests/database.ts`
```
/**
 * NOTE: This is required because globalSetup/Teardown live separately from
 * the rest of the jest ecosystem.
 *
 * This means that the files defined therein won't honour the aliases defined within
 * our jest config and we have to import using relative/absolute imports.
 */
```

I verified that this was indeed an issue by setting `../../constants` to `@constants/index` and got the following error:
```
Error: Jest: Got error running globalSetup - /Users/prestonlim/Documents/isomercms-backend/tests/setup.ts, reason: Cannot find module '@constants/index'
```

So I did a bit of research and found this [StackOverflow post](https://stackoverflow.com/a/67729228) that suggested an easy fix - which was to import `tsconfig-paths/register` at the top of the setup and teardown files.